build.sbt

```diff
@@ -29,15 +31,22 @@ libraryDependencies ++= Seq(
   "com.google.api-client" % "google-api-client" % "1.20.0" excludeAll ExclusionRule(organization = "com.google.guava"),
   "com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev53-1.20.0" excludeAll ExclusionRule(organization = "com.google.guava"),
   "com.h2database" % "h2" % "1.3.175",
-  "com.typesafe.akka" %% "akka-actor" % "2.5.9",
+  "com.typesafe.akka" %% "akka-actor" % akkaV,
+  "com.typesafe.akka" %% "akka-stream" % akkaV,
```

Comment:
It was necessary to explicitly specify the version for `akka-stream` to appease `Akka` as it apparently wants the core libraries to be of the same minor version (`2.4` or `2.5`).